### ISSUE TITLE
cmd/kfulint: add option not to fail build when test case failed

### ISSUE THREAD

**Issue by fexolm** [PieselBois/kfulint#61]
It would allow us writing tests for not yet implemented features.

**Comment by Quasilyte** [comment]
Can you be more specific?

**Comment by fexolm** [comment]
Right now i found a bug in parenthesis_checker. I have wrote a test to reproduce it. But I can not commit it, because it would fail the build. 
It would be better to add option to expect incorrect output on some cases, but not to fail the build.

**Comment by Quasilyte** [comment]
Still don't understand what you mean.
How exactly does it fail?

If you want to handle new case in checker do this:
```go
//TODO: Kind: message...
func f() ...
```
Note the double slashes instead of `///`.
When checker can handle that, remove TODO and add slash.

If you have code that breaks your linter, add it like this:
```go
/*
TODO: this makes checker panic. Uncomment when fixed.
func veryBad() ...
*/
```

**Comment by Quasilyte** [comment]
You usually fix a bug and **then** write a test that checks that it's never happens again.
I see no real benefit of skipping tests here.

**Comment by fexolm** [comment]
```//TODO: Kind: message...
func f() ...```
that case would fail on false positives.

**Comment by fexolm** [comment]
Maybe u right

**Comment by Quasilyte** [comment]
False positives should be added as a normal test case with a TODO note to remove warning directive after it's fixed.
```go
// TODO: should not trigger. Remove directive when fixed.
///Checker: blah-blah.
func f() {}
```
If false positive happens ever again, it will cause test to fail (as we removed directive at that time).

**Comment by fexolm** [comment]
Oh, ok.

**Comment by Quasilyte** [comment]
We need to document this. :)

**Comment by Quasilyte** [comment]
It can also be useful to leave issue reference near TODO, if there are any.
It could be in form of `#ID` or `issueXXX` or even a full link to github.
